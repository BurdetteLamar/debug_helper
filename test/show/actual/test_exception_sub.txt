---
DebugHelperTest::ExceptionSub (message='test_exception_sub'):
  DebugHelperTest::ExceptionSub#message: Boo!
  DebugHelperTest::ExceptionSub#cause: 
