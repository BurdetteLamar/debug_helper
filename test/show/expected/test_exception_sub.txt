---
DebugHelperTest::ExceptionSub (message='test_exception_sub'):
  message: Boo!
  cause: 
